continuous_integration/environment-3.6.yaml

```diff
@@ -22,9 +22,7 @@ dependencies:
   - zarr
   - tiledb-py
   - sqlalchemy
-  # Remove <2 version constraint once we've added pyarrow 2.0 compatibility
-  # xref https://github.com/dask/dask/issues/6754
-  - pyarrow>=0.14.0,<2
+  - pyarrow>=0.14.0,<2|>2
```

Comment:
@jrbourbeau (sorry for the basic question): Do you know if `>2` includes `2.1.0`, or does it exclude all `2.*`?

Reply:
I think it should include `2.1.0`. Locally I created a conda environment with `conda create -n test python=3.8 "dask>2"` which installed `dask=2.30.0`

Reply:
Though maybe something like `!=2.0.0` would be more clear if there's just a single version we want to exclude